docs/config.json

```diff
@@ -683,6 +683,20 @@
             {
               "title": "Standalone",
               "slug": "/kubernetes-access/guides/standalone-teleport/"
+            },
+            {
+              "title": "Kubernetes Discovery (Preview)",
```

Comment:
Since the maximum header depth our sidebar can show is three, it might make sense to put this at the same header depth as the "Guides" section, rather than a subsection of "Guides".

Reply:
Moved to `Kubernetes Access`